contracts/IERC777Recipient.sol

```diff
@@ -4,14 +4,28 @@
 
 pragma solidity ^0.5.0;
 
+/**
+* @title ERC777 Token Recipient Interface
+* @author Andrea Di Nenno
+* @notice This interface allows a token receiver (EOA or Smart Contract) to add extra logic 
+* to control his incoming tokens.
+*/
 
 interface IERC777Recipient {
+
     /**
-     * @dev Called by an `IERC777` token contract whenever tokens are being
-     * moved or created into a registered account (`to`).
+    * @notice Called by a `ERC777` token contract whenever tokens are being moved or created into a account (`to`).
+    * This call occurs after the contract state (i.e. balances) is updated and may revert to prevent transfer.
+    * It is MANDATORY for a Smart Contract that intends to receive and manage tokens to implement this interface, 
+    * or each transfer to it would revert.
+    * @param operator address of the operator, if any, that triggered the transfer
```

Comment:
Oh, we're missing the formatting for the params - operator -> `operator`

Reply:
that formatting is produced automatically with the @param keyword.